src/Microsoft.Data.DataView/SchemaDebuggerProxy.cs

```diff
@@ -41,16 +41,16 @@ public AnnotationsDebuggerProxy(DataViewSchema.Annotations annotations)
             foreach (var column in annotations.Schema)
             {
                 var name = column.Name;
-                var value = Utils.MarshalInvoke(GetValue<int>, column.Type.RawType, annotations, column.Index);
+                var value = Utils.MarshalInvoke(GetValue<DataViewSchema.Column>, column.Type.RawType, annotations, column);
```

Comment:
>DataViewSchema.Column [](start = 57, length = 21)

Incidentally this was not necessary or helpful. The point is to merely have *a* method for the method to sink its teeth into, so the type parameter here was not informative. (If anything it is now more obscure.)

Reply:
This is not essential -- the method will still work the same even with this change, and there should be no cost to it. Just pointing out that there was no need for it.

---
In reply to: [263101561](https://github.com/dotnet/machinelearning/pull/2796#discussion_r263101561) [](ancestors = 263101561)